connector_search_engine/models/se_index.py

```diff
@@ -64,11 +66,12 @@ def _jobify_batch_export(self):
         description = _("Prepare a batch export of index '%s'") % self.name
         self.with_delay(description=description).batch_export()
 
-    @api.model
+    @api.multi
     def generate_batch_export_per_index(self, domain=None):
-        if domain is None:
-            domain = []
-        for record in self.search(domain):
+        recordset = self
```

Comment:
I think there is an issue here.
Indeed the method "generate_batch_export_per_index" is called by the cron.
If the domain is None it should search for all index in order to create the job.
Here if we call with the cron (and so on the model) without domain self will be empty an nothing will be exported

Reply:
I see, my bad. Easy fix then :)